frappe/utils/background_jobs.py

```diff
@@ -34,9 +35,11 @@ def get_queues_timeout():
 	custom_workers_config = common_site_config.get("workers", {})
 	default_timeout = 300
 
+	# Note: Order matters here
+	# If no queues are specified then RQ prioritizes queues in specified order
 	return {
-		"default": default_timeout,
```

Comment:
so we gave default more priority than short, and short more than long?

now it's short > default > long ?

maybe we should give default more priority than short? - as the default in `frappe.enqueue` function is also `default` (?)

main usage of short queues are emails (afaik) - which can be considered as secondary usage as compared to other enqueued jobs (?)

Reply:
Short was always supposed to have highest priority hence a seperate identical queue as default.

So far it didn't matter cause we don't use multi-queue consumption.